src/helpers/dateTimeHelper.ts

```diff
@@ -0,0 +1,3 @@
+export function calcuateTimeDifferenceMilliseconds(startTime: Date, endTime: Date): number {
```

Comment:
```suggestion
export function calculateTimeDifferenceInMilliseconds(startTime: Date, endTime: Date): number {
```

Can we use `moment` here as well? Something like this should work, `return moment(endTime).diff(moment(startTime))`

Reply:
what would be the benefit in introducing moment for this calculation?

I had thought about it but since the dates will all be in UTC, we dont need to do anything with timezones and its such a simple calculation I thought bringing in another node module would be pointless. 

Node modules have to be zipped up with the lambda function so I was leaning towards having less to zip up

Reply:
Fair point, yeah that makes sense 👍 can we remove the `moment-timezone` dependency from package.json then?

Reply:
Yep didnt see it left in there thanks!